### PR TITLE
New Prison Station

### PR DESCRIPTION
**Pull request by Alterist**
From #260
A modification of Prison Station
Changes includes:
Moves DS2 to Civilian Hanger
Moves Xeno Spawns to the center of the map
Removes the first layer of podlocks from the western side
Adds tiny gardens that could be shoveled for dirt to LZ1 and LZ2
Fixes a few weird areas
Slightly expands Max-Sec panopticon
Removes most heavy reinforced walls within the station
Removes most 4-block walls 
Moves Telecomms off-station, marines do not have to setup comms now

Old Prison station + area fixes moved to backup
tl;dr: Xenos in middle, Marines on both sides, more strategy and options

### CODE REVIEW

**Comment by Ghommie** [comment]
MapBot is ever so quite on vacation here, mind if you post some screenies out of curiosity?

**Comment by Alterist** [comment]
colonialmarinesalpha 2018-09-23 192143
small garden added to North Civ Residences

colonialmarinesalpha 2018-09-23 192159
Pool added to South Civ Residence

colonialmarinesalpha 2018-09-23 192204
Bar added to the center of Civ Residences

colonialmarinesalpha 2018-09-23 192955
Expanded Max-Sec

colonialmarinesalpha 2018-09-23 193037
Small gardens added to LZ1

colonialmarinesalpha 2018-09-23 193746
The Central Yard, now very likely a Xeno nest

**Comment by Supermichael777** [comment]
>marines ever using DS2 to attack

**Comment by Rohesie** [comment]
You will have to make like in LV, moving comms out of the map, if you want marines to ever use LZ 2.
